end_to_end_tests/golden-record/my_test_api_client/api/default/ping_ping_get.py

```diff
@@ -6,10 +6,7 @@
 from ...types import Response
 
 
-def _get_kwargs(
-    *,
-    client: Client,
-) -> Dict[str, Any]:
+def _get_kwargs(*, client: Client,) -> Dict[str, Any]:
```

Comment:
Bummer looks like running black changed a lot of files I didn't intend to touch :/

I'll clean the commits up tomorrow.